app/views/layouts/application.html.erb

```diff
@@ -9,14 +9,11 @@
       <div class="Docs__page-container StandardWhiteContentPage">
         <div class="Docs__page-container__inner PageContainer">
           <div class="Docs__article">
-            <div class="column1">
-              <div class="TextContent">
-                <%= yield %>
-              </div>
+            <%= yield :toc %>
+            <div class="TextContent">
+              <%= yield %>
               <%= render "footer", :embed_emojicom => true %>
```

Comment:
Let's move the footer out of `.TextContent`. It's breaking the styles a bit, e.g. border around the emojicom iframe, changing he button styles.

Reply:
Good catch! 

FYI I've got a bit of another follow up CSS fix that does some larger style changes that I reckon will make this all work a bit better using CSS grids.